pkg/controller/acicontainersoperator.go

```diff
@@ -1071,7 +1071,7 @@ func (c *Controller) enableRouteInformer(stopCh <-chan struct{}) {
 		var options metav1.ListOptions
 		for {
 			Pods, err := c.K8s_Clientset.CoreV1().Pods("openshift-apiserver").List(options)
-			if err == nil && (len(Pods.Items) > 0 && Pods.Items[0].Status.Phase == v1.PodRunning) {
+			if err == nil && (len(Pods.Items) > 0 && Pods.Items[0].Status.ContainerStatuses[0].Ready == true) {
```

Comment:
do you want to check for both the container ID's ?

Reply:
this might perhaps not be necessary..picking this change for now, we can decide as follow up whether we need to check for both containers